### ISSUE TITLE
Anderson acceleration unstable

### ISSUE THREAD

**Issue by sbarratt** [cvxgrp/scs#129]
I'm getting unstable behavior with Anderson acceleration that goes away without it.

With `acceleration_lookback=10`:
```
----------------------------------------------------------------------------
	SCS v2.1.1 - Splitting Conic Solver
	(c) Brendan O'Donoghue, Stanford University, 2012
----------------------------------------------------------------------------
Lin-sys: sparse-direct, nnz in A = 26
eps = 1.00e-04, alpha = 1.50, max_iters = 5000, normalize = 1, scale = 1.00
acceleration_lookback = 10, rho_x = 1.00e-03
Variables n = 7, constraints m = 14
Cones:	primal zero / dual free vars: 2
	linear vars: 8
	soc vars: 4, soc blks: 1
Setup time: 7.92e-04s
----------------------------------------------------------------------------
 Iter | pri res | dua res | rel gap | pri obj | dua obj | kap/tau | time (s)
----------------------------------------------------------------------------
     0| 1.57e+19  1.81e+19  1.00e+00 -1.61e+19  1.58e+19  3.27e+19  5.82e-03 
   100| 1.03e+00  4.45e-01  1.38e-01 -5.05e-01 -8.28e-01  3.99e-17  6.14e-03 
   200| 3.05e-04  6.61e-04  1.16e-05 -6.40e-01 -6.40e-01  1.10e-16  6.52e-03 
   300| 1.88e-03  4.40e-03  4.32e-03 -6.23e-01 -6.33e-01  4.49e-17  6.76e-03 
   400| 2.79e-02  5.02e-02  3.51e-02 -5.42e-01 -6.18e-01  2.60e-17  7.00e-03 
   500| 7.95e-02  1.13e-01  2.04e-01 -3.90e-01 -8.46e-01  1.74e-16  7.24e-03 
   600| 1.30e+16  2.96e+16  1.00e+00  1.06e+16 -3.66e+16  2.08e+16  7.48e-03 
   700| 4.95e-01  7.93e-01  9.25e-04 -1.04e+00 -1.04e+00  1.81e-16  7.71e-03 
   800| 2.27e-03  5.54e-03  4.97e-03 -6.20e-01 -6.31e-01  1.37e-16  7.95e-03 
   900| 1.43e-02  3.51e-02  1.77e-03 -6.26e-01 -6.30e-01  4.79e-17  8.19e-03 
  1000| 1.39e+17  3.28e+17  1.00e+00  1.21e+17 -3.86e+17  2.56e+17  8.43e-03 
  1100| 3.45e-03  8.45e-03  8.13e-03 -6.09e-01 -6.28e-01  5.21e-17  8.67e-03 
  1200| 9.99e-02  2.14e-01  2.23e-02 -5.53e-01 -6.01e-01  1.14e-16  8.90e-03 
  1300| 5.01e+15  1.16e+16  1.00e+00  3.58e+15 -1.14e+16  6.79e+15  9.32e-03 
  1400| 7.65e+16  1.72e+17  1.00e+00  6.62e+16 -2.26e+17  1.26e+17  9.57e-03 
  1500| 2.67e-02  3.81e-02  2.14e-02 -6.31e-01 -5.84e-01  1.31e-17  9.82e-03 
  1600| 1.80e-01  2.59e-01  1.03e-01 -4.16e-01 -6.27e-01  6.77e-17  1.02e-02 
  1700| 9.79e-04  9.12e-04  6.98e-04 -6.38e-01 -6.40e-01  2.41e-16  1.05e-02 
  1800| 4.78e+15  1.11e+16  1.00e+00  3.70e+15 -1.09e+16  6.26e+15  1.07e-02 
  1900| 1.06e+16  2.34e+16  1.00e+00  1.11e+16 -3.17e+16  1.58e+16  1.09e-02 
  2000| 3.61e+16  8.45e+16  1.00e+00  3.06e+16 -9.93e+16  6.49e+16  1.12e-02 
  2100| 2.49e-03  6.03e-03  5.85e-03 -6.17e-01 -6.30e-01  1.29e-18  1.14e-02 
  2200| 5.04e+16  1.17e+17  1.00e+00  4.42e+16 -1.43e+17  8.60e+16  1.17e-02 
  2300| 2.58e-01  3.77e-01  5.72e-01 -7.03e-02 -1.59e+00  1.34e-16  1.19e-02 
  2400| 5.56e+16  1.30e+17  1.00e+00  4.96e+16 -1.60e+17  9.75e+16  1.21e-02 
  2500| 4.65e+16  1.08e+17  1.00e+00  4.19e+16 -1.34e+17  8.10e+16  1.24e-02 
  2600| 4.92e-02  6.92e-02  2.34e-02 -6.54e-01 -6.01e-01  6.39e-17  1.26e-02 
  2700| 2.10e-02  3.16e-02  9.53e-04 -6.39e-01 -6.41e-01  1.67e-16  1.28e-02 
  2800| 2.01e-02  2.13e-02  8.36e-03 -6.19e-01 -6.38e-01  2.86e-17  1.31e-02 
  2900| 1.86e-02  2.42e-02  6.67e-03 -6.10e-01 -5.95e-01  2.35e-17  1.36e-02 
  3000| 1.48e-01  1.84e-01  1.82e-01 -3.89e-01 -7.85e-01  0.00e+00  1.38e-02 
  3100| 6.75e+16  1.59e+17  1.00e+00  5.77e+16 -1.84e+17  1.24e+17  1.40e-02 
  3200| 8.71e+16  2.02e+17  1.00e+00  7.93e+16 -2.50e+17  1.54e+17  1.43e-02 
  3300| 9.28e-03  3.65e-03  3.73e-04 -6.39e-01 -6.40e-01  1.44e-16  1.45e-02 
  3400| 2.73e+17  6.43e+17  1.00e+00  2.28e+17 -7.13e+17  4.74e+17  1.48e-02 
  3500| 4.89e+15  1.06e+16  1.00e+00  3.48e+15 -1.02e+16  5.81e+15  1.50e-02 
  3600| 3.35e-01  4.13e-01  1.36e-01 -3.31e-01 -5.92e-01  1.04e-16  1.52e-02 
  3700| 6.54e-03  7.31e-03  6.79e-04 -6.39e-01 -6.40e-01  1.35e-16  1.55e-02 
  3800| 2.33e-01  2.84e-01  1.17e-01 -4.80e-01 -7.41e-01  1.05e-16  1.57e-02 
  3900| 1.03e-02  2.81e-02  2.47e-03 -6.28e-01 -6.33e-01  5.60e-17  1.59e-02 
  4000| 2.67e-02  5.13e-02  6.85e-02 -4.95e-01 -6.41e-01  1.00e-17  1.62e-02 
  4100| 5.80e-04  3.87e-04  7.76e-06 -6.40e-01 -6.40e-01  3.96e-17  1.65e-02 
  4200| 6.95e-02  1.02e-01  1.97e-03 -6.33e-01 -6.38e-01  1.62e-16  1.67e-02 
  4300| 3.08e-02  2.31e-02  8.36e-04 -6.27e-01 -6.25e-01  2.44e-18  1.69e-02 
  4400| 5.19e+17  1.22e+18  1.00e+00  4.46e+17 -1.52e+18  8.92e+17  1.72e-02 
  4500| 6.07e-02  1.18e-01  5.63e-03 -6.62e-01 -6.49e-01  3.81e-18  1.74e-02 
  4600| 5.70e+15  8.88e+15  1.00e+00 -1.24e+15  7.87e+14  1.31e+15  1.77e-02 
  4700| 2.41e+17  5.66e+17  1.00e+00  2.10e+17 -6.67e+17  4.45e+17  1.79e-02 
  4800| 1.03e+00  1.65e+00  4.78e-01 -1.08e+00 -5.86e-02  3.27e-16  1.82e-02 
  4900| 1.66e-02  3.82e-02  2.92e-02 -5.72e-01 -6.37e-01  5.33e-17  1.84e-02 
  5000| 2.19e-02  5.38e-02  7.45e-03 -6.10e-01 -6.27e-01  1.09e-16  1.86e-02 
----------------------------------------------------------------------------
Status: Solved/Inaccurate
Hit max_iters, solution may be inaccurate
Timing: Solve time: 1.86e-02s
	Lin-sys: nnz in L factor: 54, avg solve time: 1.35e-07s
	Cones: avg projection time: 4.07e-08s
	Acceleration: avg step time: 1.53e-06s
----------------------------------------------------------------------------
Error metrics:
dist(s, K) = 3.0145e-17, dist(y, K*) = 0.0000e+00, s'y/|s||y| = 5.5824e-20
primal res: |Ax + s - b|_2 / (1 + |b|_2) = 2.1856e-02
dual res:   |A'y + c|_2 / (1 + |c|_2) = 5.3818e-02
rel gap:    |c'x + b'y| / (1 + |c'x| + |b'y|) = 7.4539e-03
----------------------------------------------------------------------------
c'x = -0.6101, -b'y = -0.6268
============================================================================
```

With `acceleration_lookback=0`:
```
----------------------------------------------------------------------------
	SCS v2.1.1 - Splitting Conic Solver
	(c) Brendan O'Donoghue, Stanford University, 2012
----------------------------------------------------------------------------
Lin-sys: sparse-direct, nnz in A = 26
eps = 1.00e-04, alpha = 1.50, max_iters = 5000, normalize = 1, scale = 1.00
acceleration_lookback = 0, rho_x = 1.00e-03
Variables n = 7, constraints m = 14
Cones:	primal zero / dual free vars: 2
	linear vars: 8
	soc vars: 4, soc blks: 1
Setup time: 1.82e-03s
----------------------------------------------------------------------------
 Iter | pri res | dua res | rel gap | pri obj | dua obj | kap/tau | time (s)
----------------------------------------------------------------------------
     0| 1.57e+19  1.81e+19  1.00e+00 -1.61e+19  1.58e+19  3.27e+19  6.46e-03 
    60| 9.53e-05  1.08e-05  1.51e-06 -6.40e-01 -6.40e-01  8.56e-17  6.53e-03 
----------------------------------------------------------------------------
Status: Solved
Timing: Solve time: 6.54e-03s
	Lin-sys: nnz in L factor: 54, avg solve time: 1.74e-07s
	Cones: avg projection time: 4.78e-08s
	Acceleration: avg step time: 2.27e-08s
----------------------------------------------------------------------------
Error metrics:
dist(s, K) = 2.2204e-16, dist(y, K*) = 0.0000e+00, s'y/|s||y| = -3.8265e-17
primal res: |Ax + s - b|_2 / (1 + |b|_2) = 9.5281e-05
dual res:   |A'y + c|_2 / (1 + |c|_2) = 1.0761e-05
rel gap:    |c'x + b'y| / (1 + |c'x| + |b'y|) = 1.5141e-06
----------------------------------------------------------------------------
c'x = -0.6401, -b'y = -0.6401
============================================================================
```

Here's the script to reproduce:
```python
import cvxpy as cp
import numpy as np

np.random.seed(32)
n = 2
mu = np.random.randn(n)
cov_sqrt = np.random.randn(n, n)

ht = cp.Parameter(n)
ut = cp.Variable(n)
htp = cp.Variable(n)
transaction_cost = .0001 * cp.abs(ut)
objective = -mu.T @ htp + cp.sum_squares(cov_sqrt @ htp)
constraints = [
    100*cp.sum(ut) + 100*transaction_cost <= 0,
    htp == ht + ut,
    htp >= 0
]
ht.value = np.ones(n) / n
prob = cp.Problem(cp.Minimize(objective), constraints)
prob.solve(solver=cp.SCS, verbose=True)

prob = cp.Problem(cp.Minimize(objective), constraints)
prob.solve(solver=cp.SCS, verbose=True, acceleration_lookback=0)
```

Software versions:
```
python: 3.7.3
cvxpy: 1.0.25
scs: 2.1.1
```

**Comment by sbarratt** [comment]
@akshayka @junziz @bodono

**Comment by bodono** [comment]
This is fixed in the 3.0.0 branch.

**Comment by bodono** [comment]
Concretely with latest SCS 3.0.0 it takes 40625 iterations to converge to 1e-9 accuracy with `acceleration_lookback=0` but only 1800 with `acceleration_lookback=10`.

(The current plan is to disable AA by default in 3.0.0 anyway).

```
------------------------------------------------------------------
	       SCS v3.0.0 - Splitting Conic Solver
	(c) Brendan O'Donoghue, Stanford University, 2012
------------------------------------------------------------------
problem:  variables n: 7, constraints m: 14
cones: 	  f: primal zero / dual free vars: 2
	  l: linear vars: 8
	  q: soc vars: 4, soc blks: 1
settings: eps_abs: 1.0e-09, eps_rel: 1.0e-09, eps_infeas: 1.0e-07
	  alpha: 1.50, scale: 1.00e-01, adaptive_scaling: 1
	  max_iters: 100000, normalize: 1, warm_start: 0
lin-sys:  sparse-direct
	  nnz(A): 26, nnz(P): 0
------------------------------------------------------------------
 iter | pri res | dua res |   gap   |   obj   |  scale  | time (s)
------------------------------------------------------------------
     0| 1.81e+01  1.00e+00  2.22e+01 -2.28e+01  1.00e-01  5.37e-05
   250| 2.03e-04  1.54e-05  5.25e-05 -6.40e-01  1.00e-01  2.37e-04
   500| 1.88e-04  1.59e-05  5.21e-05 -6.40e-01  1.00e-01  5.33e-04
   750| 1.64e-04  1.63e-05  5.18e-05 -6.40e-01  1.00e-01  7.28e-04
  1000| 1.32e-04  1.67e-05  5.14e-05 -6.40e-01  1.00e-01  9.67e-04
  1250| 9.36e-05  1.70e-05  5.11e-05 -6.40e-01  1.00e-01  1.16e-03
  1500| 5.11e-05  1.72e-05  5.07e-05 -6.40e-01  1.00e-01  1.36e-03
  1750| 2.89e-05  1.72e-05  5.03e-05 -6.40e-01  1.00e-01  1.57e-03
  2000| 4.87e-05  1.72e-05  5.00e-05 -6.40e-01  1.00e-01  1.80e-03
  2250| 9.10e-05  1.70e-05  4.96e-05 -6.40e-01  1.00e-01  2.02e-03
  2500| 1.29e-04  1.67e-05  4.92e-05 -6.40e-01  1.00e-01  2.35e-03
  2750| 1.61e-04  1.63e-05  4.89e-05 -6.40e-01  1.00e-01  2.60e-03
  3000| 1.85e-04  1.59e-05  4.85e-05 -6.40e-01  1.00e-01  2.75e-03
  3250| 2.00e-04  1.54e-05  4.82e-05 -6.40e-01  1.00e-01  2.89e-03
  3500| 2.05e-04  1.49e-05  4.78e-05 -6.40e-01  1.00e-01  3.03e-03
  3750| 2.00e-04  1.53e-05  4.75e-05 -6.40e-01  1.00e-01  3.20e-03
  4000| 1.86e-04  1.58e-05  4.71e-05 -6.40e-01  1.00e-01  3.35e-03
  4250| 1.63e-04  1.63e-05  4.67e-05 -6.40e-01  1.00e-01  3.51e-03
  4500| 1.31e-04  1.67e-05  4.64e-05 -6.40e-01  1.00e-01  3.66e-03
  4750| 9.41e-05  1.69e-05  4.60e-05 -6.40e-01  1.00e-01  3.94e-03
  5000| 5.23e-05  1.71e-05  4.57e-05 -6.40e-01  1.00e-01  4.20e-03
  5250| 2.60e-05  1.72e-05  4.53e-05 -6.40e-01  1.00e-01  4.43e-03
  5500| 4.52e-05  1.71e-05  4.50e-05 -6.40e-01  1.00e-01  4.61e-03
  5750| 8.72e-05  1.70e-05  4.46e-05 -6.40e-01  1.00e-01  4.82e-03
  6000| 1.25e-04  1.67e-05  4.43e-05 -6.40e-01  1.00e-01  5.00e-03
  6250| 1.57e-04  1.63e-05  4.39e-05 -6.40e-01  1.00e-01  5.16e-03
  6500| 1.81e-04  1.59e-05  4.36e-05 -6.40e-01  1.00e-01  5.36e-03
  6750| 1.96e-04  1.54e-05  4.32e-05 -6.40e-01  1.00e-01  5.65e-03
  7000| 2.02e-04  1.49e-05  4.29e-05 -6.40e-01  1.00e-01  5.83e-03
  7250| 1.98e-04  1.53e-05  4.25e-05 -6.40e-01  1.00e-01  6.14e-03
  7500| 1.84e-04  1.58e-05  4.22e-05 -6.40e-01  1.00e-01  6.31e-03
  7750| 1.62e-04  1.62e-05  4.18e-05 -6.40e-01  1.00e-01  6.46e-03
  8000| 1.31e-04  1.66e-05  4.15e-05 -6.40e-01  1.00e-01  6.77e-03
  8250| 9.46e-05  1.69e-05  4.11e-05 -6.40e-01  1.00e-01  6.94e-03
  8500| 5.36e-05  1.71e-05  4.08e-05 -6.40e-01  1.00e-01  7.15e-03
  8750| 2.32e-05  1.72e-05  4.04e-05 -6.40e-01  1.00e-01  7.36e-03
  9000| 4.19e-05  1.71e-05  4.01e-05 -6.40e-01  1.00e-01  7.55e-03
  9250| 8.35e-05  1.70e-05  3.98e-05 -6.40e-01  1.00e-01  7.75e-03
  9500| 1.21e-04  1.67e-05  3.94e-05 -6.40e-01  1.00e-01  7.98e-03
  9750| 1.53e-04  1.63e-05  3.91e-05 -6.40e-01  1.00e-01  8.17e-03
 10000| 1.77e-04  1.59e-05  3.87e-05 -6.40e-01  1.00e-01  8.39e-03
 10250| 1.93e-04  1.54e-05  3.84e-05 -6.40e-01  1.00e-01  8.54e-03
 10500| 1.99e-04  1.49e-05  3.80e-05 -6.40e-01  1.00e-01  8.75e-03
 10750| 1.96e-04  1.53e-05  3.77e-05 -6.40e-01  1.00e-01  8.91e-03
 11000| 1.83e-04  1.58e-05  3.74e-05 -6.40e-01  1.00e-01  9.06e-03
 11250| 1.61e-04  1.62e-05  3.70e-05 -6.40e-01  1.00e-01  9.23e-03
 11500| 1.31e-04  1.66e-05  3.67e-05 -6.40e-01  1.00e-01  9.39e-03
 11750| 9.53e-05  1.69e-05  3.63e-05 -6.40e-01  1.00e-01  9.56e-03
 12000| 5.48e-05  1.71e-05  3.60e-05 -6.40e-01  1.00e-01  9.73e-03
 12250| 2.05e-05  1.71e-05  3.57e-05 -6.40e-01  1.00e-01  9.96e-03
 12500| 3.87e-05  1.71e-05  3.53e-05 -6.40e-01  1.00e-01  1.02e-02
 12750| 8.01e-05  1.70e-05  3.50e-05 -6.40e-01  1.00e-01  1.04e-02
 13000| 1.18e-04  1.67e-05  3.47e-05 -6.40e-01  1.00e-01  1.06e-02
 13250| 1.49e-04  1.63e-05  3.43e-05 -6.40e-01  1.00e-01  1.08e-02
 13500| 1.74e-04  1.59e-05  3.40e-05 -6.40e-01  1.00e-01  1.10e-02
 13750| 1.90e-04  1.55e-05  3.36e-05 -6.40e-01  1.00e-01  1.12e-02
 14000| 1.97e-04  1.49e-05  3.33e-05 -6.40e-01  1.00e-01  1.13e-02
 14250| 1.94e-04  1.52e-05  3.30e-05 -6.40e-01  1.00e-01  1.15e-02
 14500| 1.81e-04  1.57e-05  3.26e-05 -6.40e-01  1.00e-01  1.17e-02
 14750| 1.60e-04  1.62e-05  3.23e-05 -6.40e-01  1.00e-01  1.18e-02
 15000| 1.31e-04  1.65e-05  3.20e-05 -6.40e-01  1.00e-01  1.20e-02
 15250| 9.60e-05  1.68e-05  3.16e-05 -6.40e-01  1.00e-01  1.21e-02
 15500| 5.61e-05  1.70e-05  3.13e-05 -6.40e-01  1.00e-01  1.23e-02
 15750| 1.78e-05  1.71e-05  3.10e-05 -6.40e-01  1.00e-01  1.24e-02
 16000| 3.56e-05  1.71e-05  3.06e-05 -6.40e-01  1.00e-01  1.26e-02
 16250| 7.67e-05  1.69e-05  3.03e-05 -6.40e-01  1.00e-01  1.28e-02
 16500| 1.14e-04  1.67e-05  3.00e-05 -6.40e-01  1.00e-01  1.29e-02
 16750| 1.46e-04  1.64e-05  2.97e-05 -6.40e-01  1.00e-01  1.31e-02
 17000| 1.71e-04  1.59e-05  2.93e-05 -6.40e-01  1.00e-01  1.32e-02
 17250| 1.87e-04  1.55e-05  2.90e-05 -6.40e-01  1.00e-01  1.34e-02
 17500| 1.94e-04  1.50e-05  2.87e-05 -6.40e-01  1.00e-01  1.36e-02
 17750| 1.92e-04  1.52e-05  2.83e-05 -6.40e-01  1.00e-01  1.37e-02
 18000| 1.80e-04  1.57e-05  2.80e-05 -6.40e-01  1.00e-01  1.40e-02
 18250| 1.60e-04  1.61e-05  2.77e-05 -6.40e-01  1.00e-01  1.42e-02
 18500| 1.31e-04  1.65e-05  2.74e-05 -6.40e-01  1.00e-01  1.43e-02
 18750| 9.68e-05  1.68e-05  2.70e-05 -6.40e-01  1.00e-01  1.45e-02
 19000| 5.74e-05  1.70e-05  2.67e-05 -6.40e-01  1.00e-01  1.47e-02
 19250| 1.55e-05  1.71e-05  2.64e-05 -6.40e-01  1.00e-01  1.49e-02
 19500| 3.25e-05  1.71e-05  2.60e-05 -6.40e-01  1.00e-01  1.51e-02
 19750| 7.35e-05  1.69e-05  2.57e-05 -6.40e-01  1.00e-01  1.52e-02
 20000| 1.11e-04  1.67e-05  2.54e-05 -6.40e-01  1.00e-01  1.54e-02
 20250| 1.43e-04  1.64e-05  2.51e-05 -6.40e-01  1.00e-01  1.56e-02
 20500| 1.68e-04  1.60e-05  2.47e-05 -6.40e-01  1.00e-01  1.58e-02
 20750| 1.85e-04  1.55e-05  2.44e-05 -6.40e-01  1.00e-01  1.60e-02
 21000| 1.92e-04  1.50e-05  2.41e-05 -6.40e-01  1.00e-01  1.62e-02
 21250| 1.91e-04  1.52e-05  2.38e-05 -6.40e-01  1.00e-01  1.63e-02
 21500| 1.79e-04  1.57e-05  2.34e-05 -6.40e-01  1.00e-01  1.66e-02
 21750| 1.59e-04  1.61e-05  2.31e-05 -6.40e-01  1.00e-01  1.68e-02
 22000| 1.32e-04  1.65e-05  2.28e-05 -6.40e-01  1.00e-01  1.69e-02
 22250| 9.76e-05  1.68e-05  2.25e-05 -6.40e-01  1.00e-01  1.71e-02
 22500| 5.88e-05  1.70e-05  2.21e-05 -6.40e-01  1.00e-01  1.72e-02
 22750| 1.72e-05  1.71e-05  2.18e-05 -6.40e-01  1.00e-01  1.74e-02
 23000| 2.96e-05  1.71e-05  2.15e-05 -6.40e-01  1.00e-01  1.75e-02
 23250| 7.04e-05  1.69e-05  2.12e-05 -6.40e-01  1.00e-01  1.76e-02
 23500| 1.08e-04  1.67e-05  2.09e-05 -6.40e-01  1.00e-01  1.78e-02
 23750| 1.40e-04  1.64e-05  2.05e-05 -6.40e-01  1.00e-01  1.79e-02
 24000| 1.65e-04  1.60e-05  2.02e-05 -6.40e-01  1.00e-01  1.81e-02
 24250| 1.82e-04  1.55e-05  1.99e-05 -6.40e-01  1.00e-01  1.82e-02
 24500| 1.90e-04  1.50e-05  1.96e-05 -6.40e-01  1.00e-01  1.84e-02
 24750| 1.89e-04  1.51e-05  1.93e-05 -6.40e-01  1.00e-01  1.85e-02
 25000| 1.79e-04  1.56e-05  1.89e-05 -6.40e-01  1.00e-01  1.87e-02
 25250| 1.59e-04  1.61e-05  1.86e-05 -6.40e-01  1.00e-01  1.88e-02
 25500| 1.32e-04  1.65e-05  1.83e-05 -6.40e-01  1.00e-01  1.90e-02
 25750| 9.86e-05  1.68e-05  1.80e-05 -6.40e-01  1.00e-01  1.91e-02
 26000| 6.02e-05  1.70e-05  1.77e-05 -6.40e-01  1.00e-01  1.93e-02
 26250| 1.89e-05  1.71e-05  1.73e-05 -6.40e-01  1.00e-01  1.94e-02
 26500| 2.67e-05  1.70e-05  1.70e-05 -6.40e-01  1.00e-01  1.96e-02
 26750| 6.74e-05  1.69e-05  1.67e-05 -6.40e-01  1.00e-01  1.97e-02
 27000| 1.05e-04  1.67e-05  1.64e-05 -6.40e-01  1.00e-01  1.99e-02
 27250| 1.37e-04  1.64e-05  1.61e-05 -6.40e-01  1.00e-01  2.00e-02
 27500| 1.63e-04  1.60e-05  1.57e-05 -6.40e-01  1.00e-01  2.02e-02
 27750| 1.80e-04  1.55e-05  1.54e-05 -6.40e-01  1.00e-01  2.03e-02
 28000| 1.89e-04  1.50e-05  1.51e-05 -6.40e-01  1.00e-01  2.05e-02
 28250| 1.88e-04  1.51e-05  1.48e-05 -6.40e-01  1.00e-01  2.06e-02
 28500| 1.78e-04  1.56e-05  1.45e-05 -6.40e-01  1.00e-01  2.07e-02
 28750| 1.59e-04  1.60e-05  1.42e-05 -6.40e-01  1.00e-01  2.09e-02
 29000| 1.33e-04  1.64e-05  1.38e-05 -6.40e-01  1.00e-01  2.10e-02
 29250| 9.96e-05  1.67e-05  1.35e-05 -6.40e-01  1.00e-01  2.12e-02
 29500| 6.16e-05  1.69e-05  1.32e-05 -6.40e-01  1.00e-01  2.13e-02
 29750| 2.06e-05  1.70e-05  1.29e-05 -6.40e-01  1.00e-01  2.15e-02
 30000| 2.38e-05  1.70e-05  1.26e-05 -6.40e-01  1.00e-01  2.16e-02
 30250| 6.45e-05  1.69e-05  1.23e-05 -6.40e-01  1.00e-01  2.18e-02
 30500| 1.02e-04  1.67e-05  1.19e-05 -6.40e-01  1.00e-01  2.19e-02
 30750| 1.35e-04  1.64e-05  1.16e-05 -6.40e-01  1.00e-01  2.21e-02
 31000| 1.60e-04  1.60e-05  1.13e-05 -6.40e-01  1.00e-01  2.22e-02
 31250| 1.78e-04  1.56e-05  1.10e-05 -6.40e-01  1.00e-01  2.24e-02
 31500| 1.87e-04  1.51e-05  1.07e-05 -6.40e-01  1.00e-01  2.25e-02
 31750| 1.87e-04  1.51e-05  1.04e-05 -6.40e-01  1.00e-01  2.27e-02
 32000| 1.78e-04  1.56e-05  1.00e-05 -6.40e-01  1.00e-01  2.28e-02
 32250| 1.60e-04  1.60e-05  9.73e-06 -6.40e-01  1.00e-01  2.30e-02
 32500| 1.33e-04  1.64e-05  9.41e-06 -6.40e-01  1.00e-01  2.31e-02
 32750| 1.01e-04  1.67e-05  9.10e-06 -6.40e-01  1.00e-01  2.33e-02
 33000| 6.31e-05  1.69e-05  8.78e-06 -6.40e-01  1.00e-01  2.34e-02
 33250| 2.24e-05  1.70e-05  8.47e-06 -6.40e-01  1.00e-01  2.36e-02
 33500| 2.10e-05  1.70e-05  8.15e-06 -6.40e-01  1.00e-01  2.37e-02
 33750| 6.17e-05  1.69e-05  7.84e-06 -6.40e-01  1.00e-01  2.39e-02
 34000| 9.94e-05  1.67e-05  7.52e-06 -6.40e-01  1.00e-01  2.40e-02
 34250| 1.32e-04  1.64e-05  7.21e-06 -6.40e-01  1.00e-01  2.42e-02
 34500| 1.58e-04  1.60e-05  6.90e-06 -6.40e-01  1.00e-01  2.43e-02
 34750| 1.77e-04  1.56e-05  6.58e-06 -6.40e-01  1.00e-01  2.45e-02
 35000| 1.86e-04  1.51e-05  6.27e-06 -6.40e-01  1.00e-01  2.46e-02
 35250| 1.87e-04  1.51e-05  5.95e-06 -6.40e-01  1.00e-01  2.48e-02
 35500| 1.21e-04  1.55e-05  3.04e-06 -6.40e-01  1.00e-01  2.49e-02
 35750| 2.65e-07  6.90e-08  5.32e-08 -6.40e-01  1.00e-01  2.51e-02
 36000| 2.50e-07  1.45e-13  1.52e-11 -6.40e-01  1.00e-01  2.52e-02
 36250| 2.50e-07  1.44e-13  1.52e-11 -6.40e-01  1.00e-01  2.54e-02
 36500| 2.50e-07  1.44e-13  1.52e-11 -6.40e-01  1.00e-01  2.55e-02
 36750| 2.50e-07  1.44e-13  1.52e-11 -6.40e-01  1.00e-01  2.57e-02
 37000| 2.50e-07  1.44e-13  1.52e-11 -6.40e-01  1.00e-01  2.58e-02
 37250| 2.50e-07  1.43e-13  1.51e-11 -6.40e-01  1.00e-01  2.60e-02
 37500| 2.50e-07  1.45e-13  1.51e-11 -6.40e-01  1.00e-01  2.62e-02
 37750| 2.50e-07  1.44e-13  1.51e-11 -6.40e-01  1.00e-01  2.63e-02
 38000| 2.50e-07  1.43e-13  1.51e-11 -6.40e-01  1.00e-01  2.65e-02
 38250| 2.50e-07  1.44e-13  1.51e-11 -6.40e-01  1.00e-01  2.66e-02
 38500| 2.50e-07  1.42e-13  1.51e-11 -6.40e-01  1.00e-01  2.68e-02
 38750| 2.50e-07  1.42e-13  1.51e-11 -6.40e-01  1.00e-01  2.69e-02
 39000| 2.50e-07  1.43e-13  1.51e-11 -6.40e-01  1.00e-01  2.71e-02
 39250| 2.50e-07  1.42e-13  1.50e-11 -6.40e-01  1.00e-01  2.73e-02
 39500| 2.50e-07  1.42e-13  1.50e-11 -6.40e-01  1.00e-01  2.74e-02
 39750| 2.50e-07  1.43e-13  1.50e-11 -6.40e-01  1.00e-01  2.77e-02
 40000| 2.50e-07  1.43e-13  1.50e-11 -6.40e-01  1.00e-01  2.79e-02
 40250| 2.50e-07  1.42e-13  1.50e-11 -6.40e-01  1.00e-01  2.81e-02
 40500| 2.50e-07  6.36e-12  5.35e-12 -6.40e-01  1.57e+02  2.83e-02
 40625| 1.80e-12  1.41e-09  5.71e-10 -6.40e-01  9.29e+02  2.85e-02
------------------------------------------------------------------
status:  solved
timings: total: 2.87e-02s = setup: 1.73e-04s + solve: 2.85e-02s
	 lin-sys: 1.20e-02s, cones: 3.64e-03s, accel: 9.35e-04s
lin-sys: nnz(L): 54
------------------------------------------------------------------
cones: dist(s, K) = 0.00e+00, dist(y, K*) = 1.55e-15
comp slack: s'y/|s||y| = -8.48e-16, gap: |x'Px+c'x+b'y| = 5.71e-10
pri res: |Ax+s-b| = 1.80e-12, dua res: |Px+A'y+c| = 1.41e-09
------------------------------------------------------------------
objective = -0.640073
------------------------------------------------------------------
------------------------------------------------------------------
	       SCS v3.0.0 - Splitting Conic Solver
	(c) Brendan O'Donoghue, Stanford University, 2012
------------------------------------------------------------------
problem:  variables n: 7, constraints m: 14
cones: 	  f: primal zero / dual free vars: 2
	  l: linear vars: 8
	  q: soc vars: 4, soc blks: 1
settings: eps_abs: 1.0e-09, eps_rel: 1.0e-09, eps_infeas: 1.0e-07
	  alpha: 1.50, scale: 1.00e-01, adaptive_scaling: 1
	  max_iters: 100000, normalize: 1, warm_start: 0
	  acceleration_lookback: 10, acceleration_interval: 1
lin-sys:  sparse-direct
	  nnz(A): 26, nnz(P): 0
------------------------------------------------------------------
 iter | pri res | dua res |   gap   |   obj   |  scale  | time (s)
------------------------------------------------------------------
     0| 1.81e+01  1.00e+00  2.22e+01 -2.28e+01  1.00e-01  5.46e-05
   250| 3.27e-05  1.61e-05  5.70e-05 -6.40e-01  1.00e-01  4.24e-03
   500| 5.03e-04  1.53e-05  4.56e-05 -6.40e-01  1.00e-01  5.33e-03
   750| 8.09e-04  3.66e-05  4.64e-05 -6.40e-01  1.00e-01  6.29e-03
  1000| 2.35e-02  1.38e-03  2.33e-05 -6.40e-01  1.00e-01  6.94e-03
  1250| 4.70e-03  4.99e-05  1.31e-05 -6.40e-01  1.00e-01  7.71e-03
  1500| 2.50e-07  9.94e-13  2.96e-11 -6.40e-01  3.23e-01  8.49e-03
  1750| 2.50e-07  1.16e-09  5.92e-10 -6.40e-01  6.50e+03  9.30e-03
  1800| 8.23e-15  4.04e-10  5.83e-11 -6.40e-01  5.82e+04  9.52e-03
------------------------------------------------------------------
status:  solved
timings: total: 9.66e-03s = setup: 1.22e-04s + solve: 9.54e-03s
	 lin-sys: 6.01e-04s, cones: 2.16e-04s, accel: 7.78e-03s
lin-sys: nnz(L): 54
------------------------------------------------------------------
cones: dist(s, K) = 0.00e+00, dist(y, K*) = 6.39e-13
comp slack: s'y/|s||y| = -3.45e-13, gap: |x'Px+c'x+b'y| = 5.83e-11
pri res: |Ax+s-b| = 8.23e-15, dua res: |Px+A'y+c| = 4.04e-10
------------------------------------------------------------------
objective = -0.640073
------------------------------------------------------------------
```
